scriptoocloud/src/main/java/edu/asu/diging/scriptoocloud/web/AddProjectController.java

```diff
@@ -0,0 +1,57 @@
+package edu.asu.diging.scriptoocloud.web;
+
+import java.security.Principal;
+
+import javax.validation.Valid;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.servlet.mvc.support.RedirectAttributes;
+
+import edu.asu.diging.scriptoocloud.config.SimpleUsersConfig;
+import edu.asu.diging.scriptoocloud.core.model.Project;
+import edu.asu.diging.scriptoocloud.core.model.impl.ProjectImpl;
+import edu.asu.diging.scriptoocloud.core.service.ProjectManager;
+import edu.asu.diging.simpleusers.core.model.IUser;
+import edu.asu.diging.simpleusers.core.model.impl.SimpleUser;
+import edu.asu.diging.simpleusers.core.service.IUserManager;
+
+@Controller
+public class AddProjectController {
+
+    @Autowired
+    private ProjectManager projectManager;
+
+    @Autowired
+    private final IUserManager userManager;
+
+    @Autowired
+    public AddProjectController(IUserManager userManager) {
```

Comment:
either annotate constructor or class variables but not both (consistency)

Reply:
IUserManager annotated in STC-11, merged into STC-10 so change is now reflected.